src/server/middlewares/rendering.js

```diff
@@ -52,7 +52,7 @@ function injectJSON(data) {
   return JSON.stringify(data, null, process.env.NODE_ENV === 'production' ? 0 : 2)
 }
 
-export default (req, res) => {
+export default additionalClientData => (req, res) => {
```

Comment:
Don't we already do that with `req.user`? We could mutate the request otherwise 🙈